src/plugins/data_source_management/public/components/text_content/text_content.ts

```diff
@@ -6,278 +6,258 @@
+export const CREDENTIAL = i18n.translate('dataSourcesManagement.editDataSource.credential', {
+  defaultMessage: 'Credential',
+});
+export const AUTHENTICATION_TITLE = i18n.translate(
   'dataSourcesManagement.editDataSource.authenticationTitle',
   {
     defaultMessage: 'Authentication',
   }
 );
-export const authenticationDetailsText = i18n.translate(
-  'dataSourcesManagement.editDataSource.authenticationDetailsText',
+export const AUTHENTICATION_METHOD = i18n.translate(
+  'dataSourcesManagement.editDataSource.authenticationMethod',
   {
-    defaultMessage: 'Authentication Details',
-  }
-);
-export const authenticationDetailsDescription = i18n.translate(
-  'dataSourcesManagement.editDataSource.authenticationDetailsDescription',
-  {
-    defaultMessage: 'Modify these to update the authentication type and associated details',
+    defaultMessage: 'Authentication Method',
   }
 );
-export const endpointTitle = i18n.translate('dataSourcesManagement.editDataSource.endpointTitle', {
+export const ENDPOINT_TITLE = i18n.translate('dataSourcesManagement.editDataSource.endpointTitle', {
   defaultMessage: 'Endpoint',
 });
-export const endpointDescription = i18n.translate(
+export const ENDPOINT_DESCRIPTION = i18n.translate(
   'dataSourcesManagement.editDataSource.endpointDescription',
   {
     defaultMessage:
       'This connection information is used for reference in tables and when adding to a data source connection',
   }
 );
 
-export const cancelChangesText = i18n.translate(
+export const CANCEL_CHANGES = i18n.translate(
   'dataSourcesManagement.editDataSource.cancelButtonLabel',
   {
     defaultMessage: 'Cancel changes',
   }
 );
-export const saveChangesText = i18n.translate(
-  'dataSourcesManagement.editDataSource.saveButtonLabel',
-  {
-    defaultMessage: 'Save changes',
-  }
-);
+export const SAVE_CHANGES = i18n.translate('dataSourcesManagement.editDataSource.saveButtonLabel', {
+  defaultMessage: 'Save changes',
+});
 
-export const validationErrorTooltipText = i18n.translate(
-  'dataSourcesManagement.editDataSource.saveButtonTooltipWithInvalidChanges',
+/* Create/Edit validation */
+export const DATA_SOURCE_VALIDATION_TITLE_EXISTS = i18n.translate(
+  'dataSourcesManagement.validation.titleExists',
   {
-    defaultMessage: 'Fix invalid settings before saving.',
+    defaultMessage: 'This title is already in use',
   }
 );
 
-/* Password validation */
+/* Experimental call out text */
+export const EXPERIMENTAL_FEATURE = i18n.translate('experimentalFeature', {
+  defaultMessage: 'Experimental Feature',
+});
 
-export const dataSourceValidationOldPasswordEmpty = i18n.translate(
-  'dataSourcesManagement.validation.oldPasswordEmpty',
-  {
-    defaultMessage: 'Old password cannot be empty',
-  }
-);
-export const dataSourceValidationNewPasswordEmpty = i18n.translate(
-  'dataSourcesManagement.validation.newPasswordEmpty',
+export const EXPERIMENTAL_FEATURE_CALL_OUT_DESCRIPTION = i18n.translate(
```

Comment:
didn't find any usage of this const,
Am I missing anything, could you check all other constants

Reply:
That is being used by Yibo's code

Reply:
Done removed. i will let yibo add this code in his PR.